app/app.go

```diff
@@ -127,11 +127,11 @@ import (
 )
 
 const (
-	// mission ID for staking mission to claim airdrop
-	MISSION_ID_STAKING = 1
+	// MissionIdStaking is the mission ID for staking mission to claim airdrop
```

Comment:
Since these constants are only used within the scope of this package, we should name them to `missionIDStaking` and `missionIDVoting`.  I think "Id" should be "ID" as well.